awscli/examples/application-signals/tag-resource.rst

```diff
@@ -0,0 +1,13 @@
+**To assigns one or more tags (key-value pairs) to the specified CloudWatch resource, such as a service level objective**
+
+The following ``tag-resource`` example assigns one or more tags (key-value pairs) to the specified CloudWatch resource, such as a service level objective. ::
+
+    aws application-signals tag-resource \
+        -—resource-arn "arn:aws:application-signals:us-east-1:123456789101:slo/SLOName" \
+        -—tags '{"Key":"test","Value":"value"}'
+
+Output::
```

Comment:
```suggestion
```

Reply:
Can you remove the `Output::` line from the commands that don't have any. It's actually invalid .rst, hence the test failures.